### PR TITLE
[PORT] Ports lantern fruit & Voltaic Wine from TG

### PR DESCRIPTION
**Pull request by cowbot92**
Full credit to the original PR here - https://github.com/tgstation/tgstation/pull/79204

# Document the changes in your pull request

Adds a new plant botany can grow: The Lantern Fruit

Adds a new drink that you can distill with Lantern Fruit : Voltaic Wine

# Why is this good for the game?

Some more ways to get things via botany, because why not. Plus race specific drinks are cool.

# Testing

Will provide video of me watching paint dry if requested

# Spriting

lantern
image
image
image
drink

# Wiki Documentation

guess I'll add this later too.

# Changelog

:cl:  
rscadd: Added new fruit
rscadd: Added new drink to go with that fruit
imageadd: added some icons and images for lantern fruits and drinks
/:cl:

### CODE REVIEW

**Comment by azzzertyy** [comment]
does it light up?

**Comment by cowbot92** [comment]
> does it light up?

yes it should it contains /datum/plant_gene/trait/glow/yellow)

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2726,3 +2726,15 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	name = "Voltaic Yellow Wine"
+	description = "Electrically charged wine. Recharges ethereals, but also nontoxic."
+	boozepwr = 30
+	color = "#FFAA00"
+	quality = DRINK_GOOD
+	taste_description = "static with a hint of sweetness"
+	glass_icon_state = "wine_voltaic"
+	glass_name = "Voltaic Yellow Wine"
+	glass_desc = "Shocking, not stirred."
+
```
Comment (Therandomhoboo) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2740]: Don't know if you want to make it actually charge them or not but it doesn't do anything despite the description

Comment (cowbot92) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2740]: added it

**Review by MajManatee — APPROVED** [review]
The fog is coming? 
Wrong, Plantern.

**Review by cuackles — APPROVED** [review]

**Comment by boodaliboo** [comment]
i normally wouldnt be one to say something liket this but arent glowshrooms playing the exact same role if not better i just dont see its use i love more content either way i just wish their was content in this pr you can do with this plant besides one drink, oh its a port no complaints then good job im high

**Pull request merged**